### PR TITLE
docs(changelog) add 0.11.1 changeset

### PR DESCRIPTION
**Pull request by thibaultcha**

### CODE REVIEW

**Comment by thibaultcha** [comment]
@kinman Will you please give this a grammar review? Thank you! :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,88 @@
...
+##### Core
+
+- Fixed an edge-case where `preserve_host` would sometimes craft an upstream
+  request with a Host header from a previous client request instead of the
+  current one.
+  [#2832](https://github.com/Mashape/kong/pull/2832)
+- Ensure APIs with regex URIs are evaluated in the order that they are created.
+  [#2924](https://github.com/Mashape/kong/pull/2924)
+- Fixed a typo that prevented the load balancing components to respect
+  the Upstream `slots` property.
```
Comment (kinman) [inline CHANGELOG.md]: Fixed a typo that prevented the load balancing components from respecting the Upstream `slots` property.
--or--
Fixed a typo that caused the load balancing components to ignore the Upstream `slots` property.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,88 @@
...
+  [#2918](https://github.com/Mashape/kong/pull/2918)
+
+### Added
+
+##### Plugins
+
+- aws-lambda: Added support to forward the client request's HTTP method,
+  headers, URI, and body to the Lambda function.
+  [#2823](https://github.com/Mashape/kong/pull/2823)
+- key-auth: New `run_on_preflight` configuration option to skip authentication
```
Comment (kinman) [inline CHANGELOG.md]: "New `run_on_preflight` configuration option to control authentication."

As written, a user could quickly scan this section and assume he should set `run_on_preflight=on` to skip authentication. (Same comment on line 62.)

File: CHANGELOG.md (hunk 1 above)
Comment (thibaultcha) [reply CHANGELOG.md]: Oh my - that read so bad... Thanks! 😅

**Comment by thibaultcha** [comment]
@kinman Thank you!

**Pull request merged**